challenges/lib/src/test/java/challenges/linkedList/LinkedListTest.java

```diff
@@ -168,4 +168,42 @@ public void LinkedListTestKthFromEndToBigException() {
         ll.append(1);
         ll.kthFromEnd(10);
     }
+
+    @Test
```

Comment:
Great work with your tests! However, don't forget to check some of the base cases. What if one linked list was empty, or both?